Modules/SubModule_Population/xASL_adm_GzipAllFiles.m

```diff
@@ -1,10 +1,11 @@
-function xASL_adm_GzipAllFiles(ROOT, bFolder, bUseLinux)
+function xASL_adm_GzipAllFiles(ROOT, EXTERNAL, bFolder, bUseLinux)
 %xASL_adm_GzipAllFiles Zip files or folders
 %
 % FORMAT: xASL_adm_GzipAllFiles(ROOT, bFolder, bUseLinux)
 %
 % INPUT:
-%   x           - x structure containing all input parameters (REQUIRED)
+%   ROOT        - the root study directory (REQUIRED)
+%   EXTERNAL    - the path to the ExploreASL/External (OPTIONAL)
```

Comment:
I know that `ROOT` wasn't self-explanatory, but let's try to aim for this going ahead, so `EXTERNAL` should be `pathExternal` or `dirExternal`.

And new options should be added at the end, not in the middle ;)

Reply:
Reverted 👍